README.md

```diff
@@ -10,49 +8,66 @@
 
 ## Introduction
 
-**ferlab/postprocessing** is a bioinformatics pipeline that ...
+**ferlab/postprocessing** is a bioinformatics pipeline that recombines gvcf for family's samples in order to facilitate denovo identification.
```

Comment:
Definitely better than it was before, but I think that it is not complete enough. I think @DamienGnst would have a more precise wording than me, but it also add tags to ease variant filtering and analysis.

Reply:
ok, also @DamienGnst if you send me the diagrams I will put them here